content/en/docs/concepts/security/_index.md

```diff
@@ -3,4 +3,241 @@ title: "Security"
+   [security chaos engineering](https://glossary.cncf.io/security-chaos-engineering/),
+   where it's justified.
+
+## _Distribute_ lifecycle phase {#lifecycle-phase-distribute}
+
+- Ensure the security of the supply chain for container images you execute.
+- Ensure the security of the supply chain for the cluster and other components
+  that execute your application. An example of another component might be an
+  external database that your cloud-native application uses for persistence.
+
+To achieve this, you can:
+
+1. Scan container images and other artifacts for known vulnerabilities.
+1. Ensure that software distribution uses encryption in transit, with
+   a chain of trust for the software source.
+1. Adopt and follow processes to update dependencies when updates are
+   available, especially in response to security announcements.
+1. Use validation mechanisms such as digital certificates for supply
+   chain assurance.
+1. Subscribe to feeds and other mechanisms to alert you to security
+   risks.
+1. Restrict access to artifacts. Place container images in a
+   [private registry](/docs/concepts/containers/images/#using-a-private-registry)
+   that only allows authorized clients to pull images.
+
+## _Deploy_ lifecycle phase {#lifecycle-phase-deploy}
+
+Ensure appropriate restrictions on what can be deployed, who can deploy it,
+and where it can be deployed to.
+You can enforce measures from the _distribute_ phase, such as verifying the
+cryptographic identity of container image artefacts.
+
+When you deploy Kubernetes, you also set the foundation for your
+applications' runtime environment: a Kubernetes cluster (or
+multiple clusters).
+That IT infrastructure must provide the security guarantees that higher
+layers expect.
+
+## _Runtime_ lifecycle phase {#lifecycle-phase-runtime}
+
+The Runtime phase comprises three critical areas: [compute](#protection-runtime-compute),
+[access](#protection-runtime-access), and [storage](#protection-runtime-storage).
+
+
+### Runtime protection: access {#protection-runtime-access}
+
+The Kubernetes API is what makes your cluster work. Protecting this API is key
+to providing effective cluster security.
+
+Other pages in the Kubernetes documentation have more detail about how to set up
+specific aspects of access control. The [security checklist](/docs/concepts/security/security-checklist/)
+has a set of suggested basic checks for your cluster.
+
+Beyond that, securing your cluster means implementing effective
+[authentication](/docs/concepts/security/controlling-access/#authentication) and
+[authorization](/docs/concepts/security/controlling-access/#authorization) for API access. Use [ServiceAccounts](/docs/concepts/security/service-accounts/) to
+provide and manage security identities for workloads and cluster
+components.
+
+Kubernetes uses TLS to protect API traffic; make sure to deploy the cluster using
+TLS (including for traffic between nodes and the control plane), and protect the
```

Comment:
```suggestion
TLS (including the traffic between nodes and control plane), and protect the
```

Reply:
I wouldn't feel happy putting my name to that sentence.